### PR TITLE
update exports and version bump

### PR DESCRIPTION
**Pull request by benlower**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,12 @@
       "esm"
     ],
     "exports": {
+      ".": {
+        "import": {
+          "types": "./dist/esm/session.d.ts",
+          "default": "./dist/esm/session.js"
```
Comment (petersalas) [inline package.json:16]: IMO it'd be less surprising if `session.ts` was `index.ts`

File: package.json [hunk 2]
```diff
@@ -10,6 +10,12 @@
       "esm"
     ],
     "exports": {
+      ".": {
```
Comment (petersalas) [inline package.json:13]: This should just be 
```
".": "./src/session.ts",
```

(or `./src/index.ts` if you follow the other suggestion)

File: package.json (hunk 1 above)
Comment (benlower) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (benlower) [reply package.json]: done

File: README.md [hunk 3]
```diff
@@ -16,3 +16,41 @@ session.leaveCall();
...
+## Transcripts
+
+Transcripts are an array of transcript objects. Each transcript has the following properties:
+
+| property | type    | definition                                                                                                                         |
+| -------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| text     | string  | Text transcript of the speech from the end user or the agent.                                                                      |
+| isFinal  | boolean | True if the transcript represents a complete utterance. False if it is a fragment of an utterance that is still underway.          |
+| speaker  | Role    | Either "user" or "agent". Denotes who was speaking.                                                                                |
+| delta    | string  | Optional. If present, represents what was new in this fragment. Can be used to display transcripts in real-time along with speech. |
```
Comment (mdepinet) [inline README.md:56]: There's no delta in Transcripts.  There is however a medium, which may be "voice" or "text"

File: README.md [hunk 4]
```diff
@@ -16,3 +16,41 @@ session.leaveCall();
...
+
+When a call is started with `joinCall()`, an `UltravoxSessionState` object is returned. If we continue with the quick start code above, we can add event listeners for two events:
+
+```javascript
+state.addEventListener('ultravoxSessionStatusChanged', (event) => {
+  console.log('Session status changed: ', event.state);
+});
+
+state.addEventListener('ultravoxTranscriptsChanged', (event) => {
+  console.log('Transcripts updated: ', event.transcripts);
```
Comment (mdepinet) [inline README.md:30]: Maybe worth mentioning that the current state is available on the event also.

**Pull request merged**